IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MechanicalFastenerOverheadContactSystemFitting/DocPropertySet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MechanicalFastenerOverheadContactSystemFitting" UniqueId="aa214672-692f-ec0e-bef7-094de9f511bb" ApplicableType="IfcMechanicalFastener/COUPLER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Can we use the abbreviation in the PSet and place the full title in the Documentation?
Pset_MechanicalFastenerTypeOCSFitting